src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -122,27 +124,23 @@ public class UTIL_ACCT_Naming {
         String sortByFirstName = 'FirstName';
 
         //Build dynamic query string
-        String dynamicSoql = 'SELECT id, RecordTypeId, ';
+        String dynamicSoql = 'SELECT Id, RecordTypeId, ';
         dynamicSoql += '(SELECT Id, AccountId,' +
             'Account.RecordTypeID, ' +
             'Account.Primary_Contact__c, Account.Name,' +
             'firstname, lastname, OwnerId, Salutation,' +
             'MailingStreet, MailingCity, MailingState, ' +
             'MailingPostalCode, MailingCountry, MailingLatitude, ' +
             'MailingLongitude, OtherStreet, OtherCity, OtherState, ' +
-            'OtherPostalCode, OtherCountry, OtherLatitude, OtherLongitude, ';
+            'OtherPostalCode, OtherCountry, OtherLatitude, OtherLongitude, ' +
+            'Phone, Fax';
```

Comment:
> Can I get clarification of why these two fields moved into the query? Is it just that there wasn't a good reason not to query for them directly initially? I'm having trouble understanding why we had the if statement that has now been removed... (line 139 before)

@ceiroa -- This is a quick question with a very long answer. I will do my best to summarize :)
Problem #1: Initially, when the check for hhAccountRecordTypeId was invoked, the line adding soql was missing the closing parentheses. This is what is resulting in the current production issue.
Problem #2: When I fixed the issue naming the account based on the record type, this check was blowing up because it is building the query based on the default account model (defaultRecTypeId in the code). Therefore, if I was modifying the contacts of a HH account in an org that was set to have Admin accounts be default, the Exclude from HHName and Deceased flags weren't getting included, and therefore the HH Account wasn't being named properly as it was taking contacts who should be ignored, and including them.
Therefore, I removed that check all together. It really didn't seem to be adding much and the drawbacks seemed minimal. I also move Phone and Fax into the main query as having them only as part of the HH Account seemed odd and it made the query easier to understand.